### DISCUSSION TITLE
schakelprogamma-cmd

### DISCUSSION THREAD

**Discussion by Daniquedejong** [fdnd/docs.fdnd.nl#68] [Reflectie]
Dit is de eerste thread. Kijk voor de discussie naar de reacties van de thread!

**Comment by IvarSchuyt** [comment]
**Feedback over de koppeling tussen CMD en FDND**

- De indruk die we van te voren hadden van CMD over FDND, was puur gebaseerd op wat de docenten + ex-student ons erover verteld hadden. Toen we eenmaal aankwamen bij CMD bleken veel zaken toch anders te zitten. Dingen zoals het 2.5-jarige studietraject voelde slecht uitgelegd toen we eenmaal op CMD kwamen. Ik vind dat dit een stuk beter uitgelegd had kunnen worden.
- Ik merkte dat de communicatie tussen docenten van CMD en de docenten FDND eigenlijk niet bestaand was. De enige communicatie was tussen onze Schakelprogramma begeleider en Justus. Deze begeleider heeft ons in totaal maar een paar uur gezien tijdens het hele schakeltraject en had dus totaal geen duidelijk beeld over ons. Ik vind dat de communicatie VEEL beter had kunnen verlopen en ook meer behandeld/betrokken moet worden bij de beoordeling van het assessment.
- Er hoort een stagebezoek te zijn bij elke student in de vrije ruimte. Bij CMD is die totaal vergeten, want hij had wel plaats moeten vinden. Achteraf bij het assessment bleken docenten hier niet op de hoogte van te zijn.
- De prestaties/presentie bij CMD heeft geen invloed op je cijfer. Dit is vreemd, vooral de presentie. Ook dit moet beter gecommuniceerd worden tussen de opleidingen.

**Comment by kosterm14** [comment]
Mijn ervaring met CMD:

Het was opzich een goede opleiding maar voor mij was het veelte design heavy. Dit is dan ook de reden dat ik niet verder ga met CMD. Het schakelprogramma haalde alle code vakken weg van CMD en dit was een beetje onnodig naar mijn mening. Je gaat van heel veel code op FDND naar helemaal geen code bij CMD en dan moet ik voor mijn meesterproef opeens weer heel veel coderen. Waarom laat je me niet de code vakken volgen bij CMD zodat ik nog een beetje bij blijf. De opleidingen zijn helemaal anders en je moet maar adapten.

Ook vind ik het hele assessment na CMD onnodig. Als ik alle vakken gehaald heb bij CMD, heb ik overduidelijk voldaan aan alle criteria. Maar het is helemaal niet zo makkelijk. Hoe haal ik op CMD een 7 en 8 enzo maar dan bij mijn FDND assessment haal ik het nog maar net met 12 puntjes.

**Comment by Daniquedejong** [comment]
Tijdens de vrije ruimte van de FDND opleiding heb ik er voor gekozen om het schakelprogramma van CMD te volgen. Ik wist nog niet wat ik wilde gaan doen na de opleiding en wilde de optie hebben om hierna te kunnen doorstromen.
Een paar weken voordat het programma begon werden we bericht door Fransiska Groenland. Zij deelde informatie over de aankomende tien weken en over ons persoonlijke project. 
We waren met 6 man en werden opgesplitst in twee bestaande eerstejaars klassen van CMD. Hierbij volgden we 3 vakken met deze klassen.
Deze vakken waren gericht op het designen van nieuwe online producten. Hierbij kregen we veel les over alle ontwerpmethodes en gedragsprincipes. 

Ik heb wat nieuwe dingen geleerd over designen, maar op FDND hadden we hier ook al best veel van gehad. Het was een leuke ervaring voor tien weken, maar ik ga niet instromen. Ik vind FDND een leukere opleiding, omdat je ook daadwerkelijk je design uitvoert. Bij CMD heb je een ontwerp, waar je vervolgens niks mee doet. Dit miste ik daar en op FDND krijg ik dat wel.

**Tops CMD**

* De theorie over de ontwerpmethodes en gedragsprincipes zijn leerzaam.
* De panini's in de kantine waren lekker :)

**Tips CMD**

* Cijfers moeten eerder worden nagekeken.

**Tops FDND**

* Fijn dat er goede gedragscriteria zijn die aansluiten op het werk van CMD.

**Tips FDND**

* Laat een docent langskomen op de opleiding om met ons te kletsen.
* Meer contact hebben met docenten van CMD over werkhoudingen etc.

**Comment by SCNMC** [comment]
Mijn ervaring met het schakeltraject was prettig en interessant; ik heb veel nieuwe dingen geleerd. Ik had geen idee wat ik kon verwachten. Natuurlijk had ik door de naam van het vak een ruw idee, maar de ervaring zelf was anders. Ik heb de volgende vakken gevolgd:
- Visual Interface Design
- New Product Development
- Maatschappij en Interactie
- Project Team

Ik koos voor het schakeltraject omdat ik door deze opleiding geïnteresseerd was geraakt in het ontwerpaspect. Ik wilde hier meer over leren, dus was de keuze om het schakeltraject naar CMD te volgen een goede beslissing. Waar ik voornamelijk aan moest wennen, was het bijhouden van deliverables elke week voor deze vakken. Het waren niet per se moeilijke opdrachten, maar het voelde als "meer" in vergelijking met het AD-programma, waar je in sprints aan opdrachten werkt. Hier werkte ik aan 3 vakken en sloot dit af met een project in een blok. Uiteindelijk vond ik een manier om dit beter in te plannen en ging dit goed.

Ik vond het fijn om meer principes te leren over vormgeving, vooral bij Visual Interface Design, waar ik leerde om procesmatig tot een ontwerp te komen. Verder heb ik gemerkt dat bij CMD je leert om veel onderzoek vooraf te doen; je begint niet direct met ontwerpen in software. Dit was voor mij even wennen omdat ik het leuk vind om meteen in ontwerpsoftware te duiken. Wel heb ik geleerd dat dit een cruciaal onderdeel is van de opleiding en dat je hier niet aan ontkomt.

Als ik zou moeten omschrijven wat ik hier geleerd heb, zou het onderzoek doen, vormgevingsprincipes leren en toepassen, en samenwerken zijn. Dit laatste is ook heel belangrijk. Ik moest in 2 van de 3 vakken en in het project samenwerken. Je leert dus ook jouw ideeën met die van jouw teamgenoten te combineren en echt samen als groep tot een eindproduct te komen.

Het CMD schakeltraject was dus heel leerzaam, al was het wel even wennen in het begin. Je leert gaandeweg de werkwijze beter kennen. Ik zou dit zeker aanraden als je meer wilt leren over design, verwacht in ieder geval niet dat je veel gaat coderen.
